js/msp/MSPHelper.js

```diff
@@ -192,46 +193,63 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 break;
 
             case MSPCodes.MSP_VOLTAGE_METER_CONFIG:
-                VOLTAGE_METER_CONFIGS = [];
-                var voltage_meter_count = data.readU8();
-
-                for (var i = 0; i < voltage_meter_count; i++) {
-                    var subframe_length = data.readU8();
-                    if (subframe_length != 5) {
-                        for (var j = 0; j < subframe_length; j++) {
-                            data.readU8();
+                if (semver.lt(CONFIG.apiVersion, "1.36.0")) {
```

Comment:
We probably need to check here for the firmware being Betaflight and not cleanflight, not to break compatibility. @hydra would know more.